docs/reference/configuration/keywords.rst

```diff
@@ -128,44 +128,37 @@ Commonly used keywords
 .. _data_file:
 .. topic:: DATA_FILE
 
-        Meant to be set to the filepath of an eclipse simulator input, when such
-        a simulator is used. This does two things. First, the DATA_FILE will be
-        templated, see :ref:`RUN_TEMPLATE <run_template>`. Second, ert will look
-        for the PARALLEL keyword in this file in order to set :ref:`NUM_CPU <num_cpu>`.
+    Specify the filepath to the ``.DATA`` file of Eclipse/flow.
```

Comment:
nit-picking: I believe `ECLIPSE` is usually capitalised and I would also use `OPM Flow` instead of flow

Reply:
Would you mind using the `Suggest` feature in GitHub so that I can just commit your proposed changes directly from GitHub?